plugins/coverimage.koplugin/main.lua

```diff
@@ -259,7 +272,41 @@ function CoverImage:addToMainMenu(menu_items)
                 end,
                 sub_item_table = {
                     {
-                        text = _("Scale, black background"),
+                        text_func = function()
+                            return T(_("Elastic stretch limit (%1%)"), self.cover_image_stretch_limit )
+                        end,
+                        help_text_func = function()
+                            return T(_("If the aspect ratios of the image and the screen differ less than the selected limit (%1%), the image gets stretched (instead of scaled) to fullscreen."), self.cover_image_stretch_limit )
+                        end,
+                        keep_menu_open = true,
+                        callback = function(touchmenu_instance)
+                            local old_stretch_limit = self.cover_image_stretch_limit
+                            local SpinWidget = require("ui/widget/spinwidget")
+                            local size_spinner = SpinWidget:new{
+                                width = math.floor(Device.screen:getWidth() * 0.6),
+                                value = old_stretch_limit,
+                                value_min = 0,
+                                value_max = 25,
+                                default_value = 95,
```

Comment:
That doesn't make sense, given the default at 10 above ;).

Reply:
(Talking about the default, specifically ^^. The min is fine, and I'd fine with the max).